redaxo/src/core/lib/be/page.php

```diff
@@ -199,6 +201,10 @@ public function setItemAttr($name, $value)
      * @param string      $default
      *
      * @return string|array Attribute value for given `$name` or attribute array if `$name` is `null`
+     *
+     * @template T as ?string
+     * @psalm-param T $name
+     * @psalm-return (T is string ? string : array<string, string>)
```

Comment:
Hier brauchts ein zusätzliches phpstan-return da diese psalm notation nicht bekannt ist

Reply:
Eigentlich meckert phpstan doch eher über die `@template`-zeile. Das problem hatten wir doch bei dir letztens auch, dass phpstan scheinbar dort keine zusammengesetzten typen (hier `?string`) kann.
ich hatte lokal schon getestet, wenn ich `@phpstan-template T` ohne typ setze, kommt kein fehler mehr. 
die return-zeile scheint phpstan also zu ignorieren.

Reply:
ggf. kannst du aus dem beispiel nen phpstan repro bauen und reporten?

Reply:
Naja, der Repro ist sehr einfach: https://phpstan.org/r/1a0b170b-1785-4e5f-80ca-0026ba3c55dc
Phpstan kann scheinbar hinter `of`/`as` nur Klassennamen.
Es wurde deswegen ja extra ein Check eingebaut, den wir hier sehen, und der besagt, dass die anderen Typen "not supported" sind.
https://github.com/phpstan/phpstan-src/commit/e96d1e887746e48bf118880a85ef809ea984aa5c

Reply:
ahh hatte ich schon wieder verdrängt :)